You can the failure logs from the CI builds (Details ... [Artifacts](https://dev.azure.com/vcpkg/public/_build/results?buildId=81539&view=artifacts&pathAsName=false&type=publishedArtifacts
) ... hover the failure logs artifact ... find the three dots menu on the right side ... Download.

For x64-linux:
~~~
CMake Error at /agent/_work/1/s/scripts/buildsystems/vcpkg.cmake:829 (_find_package):
  Could not find a package configuration file provided by "glslang" with any
  of the following names:

    glslangConfig.cmake
    glslang-config.cmake

  Add the installation prefix of "glslang" to CMAKE_PREFIX_PATH or set
  "glslang_DIR" to a directory containing one of the above files.  If
  "glslang" provides a separate development package or SDK, be sure it has
  been installed.
Call Stack (most recent call first):
  /agent/_work/1/s/ports/shaderc/glslang/CMakeLists.txt:3 (find_package)

--- a/ports/glslang/install-to-datadir.patch
+++ b/ports/glslang/install-to-datadir.patch
@@ -1,0 +1,38 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index b581c84..a011686 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -375,13 +375,13 @@ if(ENABLE_GLSLANG_INSTALL)
+         include("@PACKAGE_PATH_EXPORT_TARGETS@")
+     ]=])
+     
+-    set(PATH_EXPORT_TARGETS "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}/glslang-targets.cmake")
++    set(PATH_EXPORT_TARGETS "${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME}/glslang-targets.cmake")
+     configure_package_config_file(
+         "${CMAKE_CURRENT_BINARY_DIR}/glslang-config.cmake.in"
+         "${CMAKE_CURRENT_BINARY_DIR}/glslang-config.cmake"
+         PATH_VARS
+             PATH_EXPORT_TARGETS
+-        INSTALL_DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}
++        INSTALL_DESTINATION ${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME}
+     )
+     
+     write_basic_package_version_file("${CMAKE_CURRENT_BINARY_DIR}/glslang-config-version.cmake"
+@@ -392,7 +392,7 @@ if(ENABLE_GLSLANG_INSTALL)
+     install(
+         EXPORT      glslang-targets
+         NAMESPACE   "glslang::"
+-        DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}"
++        DESTINATION "${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME}"
+     )
+     
+     install(
+@@ -400,6 +400,6 @@ if(ENABLE_GLSLANG_INSTALL)
+             "${CMAKE_CURRENT_BINARY_DIR}/glslang-config.cmake"
+             "${CMAKE_CURRENT_BINARY_DIR}/glslang-config-version.cmake"
+         DESTINATION
+-            "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}"
++            "${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME}"
+     )
+ endif()
+\ No newline at end of file

--- a/ports/glslang/portfile.cmake
+++ b/ports/glslang/portfile.cmake
@@ -8,6 +8,7 @@ vcpkg_from_github(
   HEAD_REF master
   PATCHES
     ignore-crt.patch
+    install-to-datadir.patch
 )
 
 vcpkg_find_acquire_program(PYTHON3)

--- a/ports/glslang/vcpkg.json
+++ b/ports/glslang/vcpkg.json
@@ -4,6 +4,7 @@
   "port-version": 1,
   "description": "Khronos reference front-end for GLSL and ESSL, and sample SPIR-V generator",
   "homepage": "https://github.com/KhronosGroup/glslang",
+  "license": "Apache-2.0 AND BSD-3-Clause AND MIT AND GPL-3.0-or-later",
   "dependencies": [
     {
       "name": "vcpkg-cmake",

--- a/ports/shaderc/fix-tbuiltinresource-for-glslang-11-12.patch
+++ b/ports/shaderc/fix-tbuiltinresource-for-glslang-11-12.patch
@@ -1,0 +1,20 @@
+diff --git a/libshaderc_util/src/resources.cc b/libshaderc_util/src/resources.cc
+index d64e47f..6c662d3 100644
+--- a/libshaderc_util/src/resources.cc
++++ b/libshaderc_util/src/resources.cc
+@@ -125,6 +125,15 @@ const TBuiltInResource kDefaultTBuiltInResource = {
+     /* .maxTaskWorkGroupSizeY_NV = */ 1,
+     /* .maxTaskWorkGroupSizeZ_NV = */ 1,
+     /* .maxMeshViewCountNV = */ 4,
++    /* .maxMeshOutputVerticesEXT = */ 256,
++    /* .maxMeshOutputPrimitivesEXT = */ 256,
++    /* .maxMeshWorkGroupSizeX_EXT = */ 128,
++    /* .maxMeshWorkGroupSizeY_EXT = */ 128,
++    /* .maxMeshWorkGroupSizeZ_EXT = */ 128,
++    /* .maxTaskWorkGroupSizeX_EXT = */ 128,
++    /* .maxTaskWorkGroupSizeY_EXT = */ 128,
++    /* .maxTaskWorkGroupSizeZ_EXT = */ 128,
++    /* .maxMeshViewCountEXT = */ 4,
+     /* .maxDualSourceDrawBuffersEXT = */ 1,
+     // This is the glslang TLimits structure.
+     // It defines whether or not the following features are enabled.

--- a/ports/shaderc/portfile.cmake
+++ b/ports/shaderc/portfile.cmake
@@ -12,6 +12,8 @@ vcpkg_from_github(
         fix-build-type.patch
         fix-install-shaderc_util.patch
         fix-export-cmakefiles.patch
+        # NOTE: This should be removed when shaderc gets updated to use glslang 11.12.0
+        fix-tbuiltinresource-for-glslang-11-12.patch
 )
 
 file(COPY "${CMAKE_CURRENT_LIST_DIR}/build-version.inc" DESTINATION "${SOURCE_PATH}/glslc/src")

--- a/versions/g-/glslang.json
+++ b/versions/g-/glslang.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "a453755cd3b3e7c8dafec89688404269b0c40690",
+      "git-tree": "068b8919de1351c408446373cf5a30e52d07451b",
       "version": "11.12.0",
       "port-version": 1
     },

--- a/versions/s-/shaderc.json
+++ b/versions/s-/shaderc.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "d93e78fd464f2f378c8faf94e40e16410e1c41e0",
+      "git-tree": "b33d795ee7bc7bb9a02f904dc4d8e08e5f5f4900",
       "version": "2021.1",
       "port-version": 3
     },

